doc/configoptions.md

```diff
@@ -171,43 +112,20 @@ The options here are environment variables used by `mk_image`. They are typicall
 
   [only with liveeval=1] special czech version
 
-- extramod=&lt;module&gt;
-
-  add &lt;module&gt; to initrd; only _one_ module allowed (useful for testing only)
-
 
 ## boot
 
 - boot=small|hd|large|isolinux
 
-  the boot image type we should create; small is a 1.44MB image, hd for
-  boot CD with hd emulation, isolinux for a CD with 'no emulation'; large is
+  the boot image type we should create; `small` is a 1.44MB image, `hd` for
+  boot CD with hd emulation, `isolinux` for a CD with 'no emulation'; `large` is
```

Comment:
This is historical. There's nothing to set anymore.

Reply:
I see, it is still used in `bin/mk_boot` but that is not used anywhere. Delete it?

Reply:
I think so. We don't make anything 'bootable' anymore.